### DISCUSSION TITLE
Vexana 4.0.0

### DISCUSSION THREAD

**Discussion by VB10** [VB10/vexana#84] [General]
We're working on new release with @MehmetKaranlik. So if you have a any idea please share to us.

BTW. The new version i'll add many unit test with coding refactoring. I'll try to better version for network manager layer and core functionality.

**Comment by behzodfaiziev** [comment]
@VB10 That's great :), could you please also review/add  PR #82 in a new version, it would be great to get onSendProgress as a returned function

**Reply by VB10** [reply]
I'll check asap thank uuu 🌟
